environments.json

```diff
@@ -5019,7 +5019,7 @@
     "note_id": "graalvm-node-mode",
     "note_html": "Executed in Node.js/JVM mode via <code>graalvm/bin/node --jvm</code>.",
     "release": "2021-01-19",
-    "equals": "graalvm19",
```

Comment:
Because they were redundant, they only pointed to the previous version in the line of GraalVM releases.

Newer versions of an engine already inherit the results of previous versions so such `"equals"` annotations are not needed. In fact, they are never used like this in the `environments.json` file. Instead, they are used when one version of an engine/browser is to be marked as equivalent to a version of a different engine/browser (usually because they are based on the same JavaScript implementation).